x/programs/runtime/runtime.go

```diff
@@ -82,6 +82,18 @@ func (r *WasmRuntime) Initialize(ctx context.Context, programBytes []byte) (err
 	}
 
 	link := Link{wasmtime.NewLinker(r.store.Engine)}
+
+	if r.cfg.testingOnlyMode {
+		// setup wasi for debugging
+		wasiConfig := wasmtime.NewWasiConfig()
+		wasiConfig.InheritStderr()
+		wasiConfig.InheritStdout()
+		r.store.SetWasi(wasiConfig)
```

Comment:
Do we drop these calls to log on the same prod program? or does it need to be modified?

Reply:
followup

Reply:
https://github.com/ava-labs/hypersdk/issues/552